templates/network/schema.capnp

```diff
@@ -7,4 +7,5 @@ struct Schema {
     driver @3: Text; # will ensure kernel module is loaded and interfaces are up @optional
     testIps @4: List(Text); # a list of test ip used by the monitor routine, network assumed broken if not pingable
     mtu @5: Int32; # mtu of backplane optional (default 9000)
+    mode @6: Text; # optional configuration mode ovs, or native (default ovs)
```

Comment:
add a default value for the mode
```suggestion
    mode @6: Text='ovs'; # optional configuration mode ovs, or native (default ovs)
```

Reply:
we could also have a enum for the different mode and make mode of the type of the enum.

Reply:
I didn't know that default values are supported in capnp